ted/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -243,6 +239,14 @@ public double[] getAbsoluteEncoderOffsets(){
       backRightModule.getAbsoluteEncoderOffset()};
   }
 
+  /**
+   * Method to get the current acceleration reduction factor
+   * @return - the current acceleration reduction factor
+   */
+  public double getAccelerationReductionFactor() {
```

Comment:
So it seems the intent behind DrivetrainAccelerationSubsystem is to be able to call this method (and the set below) and more importantly, to provide an interface to protect access here.  If the latter is true I suggest we make these **protected**.  This would limit callers to these methods to other subsystems only (members of the package).